tests/integration/test_xrdsst.py

```diff
@@ -17,15 +17,27 @@
 from xrdsst.controllers.token import TokenController
 from xrdsst.main import XRDSSTTest
 
+
+# Waits until boolean function returns True within number of retried delays or raises error
+def waitfor(boolf, delay, retries):
+    count = 0
+    while count < retries and not boolf():
+        time.sleep(delay)
+        count += 1
+
+    if count >= retries:
+        raise Exception("Exceeded retry count " + retries + " with delay " + delay)
```

Comment:
After setting auto-configuration options in center.ini for my central server running in an LXD container, I get the a timeout and maximum retries of 300 exceeded when running "make test-all"

Reply:
What time limit works for your setup and was central server restarted?

Reply:
> What time limit works for your setup and was central server restarted?

Seems not to work at all. I have restarted my CS instance, set retries to 600. Does it work for you?

Reply:
> What time limit works for your setup and was central server restarted?

If it works for you, I will try to make it work as well, but you can go ahead and merge.

Reply:
Yes it does, too well even, as one code section that raised exception when retry limit was exceeded had been never reached, so it had a missing typecast there. Let me know if problem persists and let's sort this out.

Merging.